products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,224 @@
+this command:
+
+```bash
+go build -o "C:\Program Files\sailpoint\sail.exe"
+```
+
+Then add this directory to the system PATH parameter: 
+
+```
+C:\Program Files\sailpoint
+```
+
+You can find instructions on how to do this [here](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53).
+You only need to do this the first time you install the CLI.
+
+After setting your environment variable, close all instances of your PowerShell
+or Command Prompt, open a new instance, and make sure you can run the `sail`
+command.
+
+```shell
+sail
+```
+
+## Configuration
+
+To configure the CLI to connect and authenticate to your IDN tenant, you must do the following:
+
+- Find your tenant name. To learn how to find it, refer to [Getting Started](https://developer.sailpoint.com/idn/api/getting-started#find-your-tenant-name). The CLI will use this tenant name to connect to your IDN instance. 
+- Choose an authentication method:
+  - PAT: Create a personal access token (PAT). Make sure to note the "client ID" and "client secret." The CLI needs this information to authenticate successfully. To learn how to create a PAT, refer to [Personal Access Tokens](https://developer.sailpoint.com/idn/api/authentication#personal-access-tokens). 
+  - OAuth: Create an OAuth client. Make sure to to note the "client ID" and "client secret," as well as the "redirect port" and "redirect path." The CLI needs all this information to authenticate successfully. To learn how to create an OAuth client, refer to [Authentication Details](https://developer.sailpoint.com/idn/api/authentication/#authentication-details). 
+
+### Assisted configuration
+
+Once you have created the authentication method, you can use the `configure` command for your chosen method to configure the CLI for your tenant. 
+
+This command creates a configuration file in your home directory to store your tenant's connection details.
+
+If you are using a **PAT**, run this command: 
+
+```shell
+sail configure pat
+```
+
+[configure PAT](./assets/img/vhs/configure/configure-pat.gif)
+
+If you are using an **OAuth client**, run this command: 
+
+```shell
+sail configure oauth
+```
+
+[configure OAuth](./assets/img/vhs/configure/configure-oauth.gif)
+
+### Manual configuration
+
+Alternatively, you can manually create a configuration file in your home
+directory.
+
+On **Linux and MacOS**, open your terminal app and run these commands: 
+
+```shell
+mkdir ~/.sailpoint
+touch ~/.sailpoint/config.yaml
+```
+
+On **Windows**, run Powershell as an administrator and run these commands:
+
+```powershell
+New-Item -ItemType Directory -Path 'C:\Users\<username>\.sailpoint'
+New-Item -ItemType File -Path 'C:\Users\<username>\.sailpoint\config.yaml'
+```
+These commands will create a `config.yaml` file you can use to set your environment variables:
+
+The `config.yaml` file must contain the following information:
+
+```yaml
```

Comment:
This configuration file structure has been updated on the CLI to reflect the new one

Reply:
Okay I copied in the new config.yaml file from the updated readme.